text/0000-utilities-profile.md

```diff
@@ -0,0 +1,42 @@
+- Start Date: 2015-12-03
+- RFC PR: (leave this empty)
+- Refract Issue: (leave this empty)
+
+# Summary
+
+This proposes we move `extend`, `select`, and `option` from the base
+specification to its own profile.
+
+Note that I'm calling this a "profile" versus a "namespace." This is pending
+this [RFC](https://github.com/refractproject/rfcs/pull/30). If we do not go the
+route proposed there, we can refer to this as a Utility Namespace.
+
+# Motivation
+
+The main motivation is to make the base specification as simple as possible. In
+ever implementation we've created for consuming Refract, such as from the Parse
+Result or API Description results, we have not needed to use any of these
+elements.
+
+In light of simplifying, this keeps the base specification concerned with
+primitive, meta elements, and addressability.
+
+# Detailed design
+
+This proposal is to move `extend`, `select`, and `option` to its own
```

Comment:
Yes, I think the only thing required would be to say that the Data Structure namespace uses this other list of elements.